fem/bilininteg.hpp

```diff
@@ -3311,6 +3317,7 @@ class DGTraceIntegrator : public BilinearFormIntegrator
    VectorCoefficient *u;
    real_t alpha, beta;
    // PA extension
+   std::unique_ptr<FaceQuadratureSpace> qspace[2];
```

Comment:
Given that constructing `FaceQuadratureSpace` is now cheap (with the offset compression and caching in `Mesh` of the face info arrays), do we need to cache these, or can we just create them each time `AssemblePA` is called?

Reply:
If so, then we can remove `BilinearFormIntegrator::Update` and associated changes.

Reply:
Is the uncompressable case important for your application? If not, I would lean towards the simpler implementation.